acceptance/features/editing_standalone_page_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative '../spec_helper'
+
+feature 'Edit standalone pages' do
+  let(:editor) { EditorApp.new }
+  let(:service_name) { generate_service_name }
+  let(:cookies_heading) { 'Updated cookies heading' }
+
+  background do
+    given_I_am_logged_in
+    given_I_have_a_service
+  end
+
+  scenario 'editing footer standalone pages' do
+    given_I_have_footer_pages
+    and_I_edit_the_cookies_page
+    and_I_change_the_page_heading(cookies_heading)
+    when_I_save_my_changes
+    then_I_should_see_the_new_cookies_heading
+  end
+
+  def given_I_have_footer_pages
+    editor.footer_pages_links.click
+  end
+
+  def and_I_edit_the_cookies_page
+    editor.cookies_link.click
+  end
+
+  def and_I_change_the_page_heading(heading)
+    editor.page_heading.set(heading)
+  end
```

Comment:
I think this method already exists https://github.com/ministryofjustice/fb-editor/blob/main/acceptance/features/edit_confirmation_page_spec.rb#L35
If is the same behaviour, can we move to the common steps module?